src/controller/agent.governance.ts

```diff
@@ -145,8 +145,8 @@ export class AgentGovernance {
      */
     public async invokeHandler(agentUrl: string, agentId: string, adminApiKey: string, route: string, topic: string,
                                body: any, token?: string) : Promise<any> {
-        let result: boolean = false;
-        // tslint:disable-next-line:forin
+        let result = false;
+        // eslint-disable-next-line @typescript-eslint/no-for-in-array
```

Comment:
Can we delete this rule? I don't understand why it doesn't like `for in` arrays, they seem perfectly fine to me.